dlt/common/normalizers/naming/naming.py

```diff
@@ -31,15 +39,13 @@ def normalize_table_identifier(self, identifier: str) -> str:
         """Normalizes and shortens identifier that will function as a dataset, table or schema name, defaults to `normalize_identifier`"""
         return self.normalize_identifier(identifier)
 
-    @abstractmethod
     def make_path(self, *identifiers: str) -> str:
```

Comment:
imho these should be called split_path and join_path (same as string operations). not really important but it makes it much clearer what happens here.